packages/consumption/test/modules/identityMetadata/local/IdentityMetadata.test.ts

```diff
@@ -0,0 +1,17 @@
+import { JSONWrapper } from "@js-soft/ts-serval";
+import { CoreAddress, CoreId } from "@nmshd/core-types";
+import { IdentityMetadata } from "../../../../src";
+
+describe("IdentityMetadata", function () {
+    test("creates an object with CoreAddress as a scope", function () {
+        const localNotification = IdentityMetadata.from({
+            id: CoreId.from("anId"),
```

Comment:
Is this the `id` of the IdentityMetadata? Shouldn't it start with `IDM` then?

Reply:
not relevant in this test.

Reply:
I just deleted that dumb test